python/pyspark/streaming/context.py

```diff
@@ -46,9 +46,13 @@ def __init__(self, sparkContext, batchDuration=None, jssc=None):
         """
         Create a new StreamingContext.
 
-        :param sparkContext: :class:`SparkContext` object.
-        :param batchDuration: the time interval (in seconds) at which streaming
-                              data will be divided into batches
+        Parameters
+        ----------
+        sparkContext : :class:`SparkContext`
+            SparkContext object.
+        batchDuration : int, optional
+            the time interval (in seconds) at which streaming
+            data will be divided into batches
```

Comment:
Shall we move this to class doc instead, so it is visible in the rendered docs?

Reply:
Sounds good. Thanks!!